tests/test-recipes/metadata/_overlinking_detection/bld.bat

```diff
@@ -0,0 +1,4 @@
+@echo on
+echo int main() { return 0; }>main.c
+cl -c -EHsc -GR -Zc:forScope -Zc:wchar_t -Fomain.obj main.c
+link -out:%PREFIX%\Library\bin\main.exe main.obj
```

Comment:
This is a test artifact that shouldn't be committed

Reply:
I'l remove them